### DISCUSSION TITLE
Перевод Ideology

### DISCUSSION THREAD

**Discussion by Ainzerhald** [Ludeon/RimWorld-ru#1244] [General]
Я хотел бы спросить, когда будет русификатор DLC Ideology?

**Comment by Elevator89** [comment]
Не раньше, чем мы это DLC переведём ).
Конкретных сроков назвать не могу. Раньше, чем через 2 месяца полного перевода ждать не стоит.

**Reply by muturgan** [reply]
> Раньше, чем через 2 месяца полного перевода ждать не стоит.

Просто хотел уточнить) Скажите пожалуйста, не появилось ли понимания сроков?

**Reply by Elevator89** [reply]
@muturgan Сейчас Идеологии готовы на 60%. Работа идёт медленно, но верно. 

Сроки давать очень сложно. Оба активных переводчика работают только в свободное время, а его наличие и количество почти никогда нельзя точно предсказать.

**Reply by muturgan** [reply]
@Elevator89 понял) спасибо!
